es6.js

```diff
@@ -47,8 +47,9 @@ bom:for(;;)break﻿bom;
 // line terminators
 lineFeed:0
 0;
-carriageReturn:0
0;
-carriageReturnLineFeed:0
+carriageReturn:0
+0;
+carriageReturnLineFeed:0
```

Comment:
Let’s add an empty `.editorconfig` file to prevent this from happening again? Reduces friction for contributors. #11